app/src/main.js

```diff
@@ -135,6 +138,16 @@ app.on('ready', () => {
   }
 });
 
+app.on('ready', () => {
```

Comment:
Is this the right place to do this? I was able to get this working, but not sure if this is correct 😅

Reply:
Not sure either, you should check the docs for the package and/or file an issue there. Also, does that mean it will run on each `ready` at startup?!

Reply:
https://github.com/MarshallOfSound/electron-devtools-installer/issues/96